assets/css/style.scss

```diff
@@ -34,10 +34,14 @@ main.page-content {
   color: #111;
 }
 
+.table-borderless, .table-borderless *, .table-borderless * * {
```

Comment:
Won't `.table-borderless *` match any element under `.table-borderless` (including the ones matched by `.table-borderless * *`)?

Reply:
I'm pretty sure `*` only matches one level deep.

Reply:
You aren't thinking of `.table-borderless > *`?